### PR TITLE
Take advantage of the new README features

### LINKED ISSUE #3550: emoji on topicos (titles)
This week @github company announced the new way of rendering the **README**, where the menu is generated automatically (table of content), in the shared gif (example repository https://twitter.com/github/status/1376636651963842562) the topics had emoji and in my opinion gives a more friendly look to the project (especially in the menu)

What is your opinion about using emoji.

ezgif com-gif-maker

### PR DESCRIPTION
**Pull request by Joakker**
Closes #3550 
This PR takes advantage of the new Github features for auto generating the table of contents. It also adds emojis to every header in the document to make them distinguishable from one another

### CODE REVIEW

**Comment by avelino** [comment]
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body links content check:
- godoc.org or pkg.go.dev: True
- goreportcard.com: False
- coverage: False

> your project is about to be approved, it's under revision, _it may take a few days_

**Comment by Joakker** [comment]
I should also mention that the headers don't exactly map out to the emojis' meanings, since some of them were a bit too specific to find a suitable one, and some of them are duplicated, but in general, I tried to make the emoji be at least tangentially related in meaning to the title they correspond to.

File: README.md [hunk 1]
```diff
@@ -1,118 +1,21 @@
...
 **We have no monthly cost**_, but we have employees **working hard** to maintain the Awesome Go, with money raised we can repay the effort of each person involved! All billing and distribution will be open to the entire community._
 
 A curated list of awesome Go frameworks, libraries and software. Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 
-### Contributing
+### 🤝 Contributing
 
 Please take a quick gander at the [contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md) first. Thanks to all [contributors](https://github.com/avelino/awesome-go/graphs/contributors); you rock!
 
 #### *If you see a package or project here that is no longer maintained or is not a good fit, please submit a pull request to improve this file. Thank you!*
```
Comment (avelino) [inline README.md:17]: The content list must be exist, it will be used on the site awesome-go.com

**Review by avelino — CHANGES_REQUESTED** [review]

**Comment by Joakker** [comment]
There, this seems to work well

**Review by avelino — CHANGES_REQUESTED** [review]
Put the emoji in the prefix is breaking the test alphabetically, we have put at the end of the title

Check Travis-CI: https://travis-ci.com/github/avelino/awesome-go/builds/222335165

**Comment by avelino** [comment]
@Joakker many links are broken, are you aware of this?

Can you keep working on this change?

**Comment by Joakker** [comment]
I can. But most of the links work on my end, so I don't know what is going on

**Pull request closed without merging**